addons/hr_recruitment_survey/security/hr_recruitment_survey_security.xml

```diff
@@ -1,16 +1,135 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data noupdate="1">
-        <record id="hr_recruitment.group_hr_recruitment_interviewer" model="res.groups">
-            <field name="implied_ids" eval="[Command.link(ref('survey.group_survey_user'))]"/>
-            <field name="comment">It will also allow to send surveys and see the resume.</field>
+        <!--
+        Specific survey access rules for recruitment
+        - The recruitment manager can CRUD survey / questions / question answers for survey_type == 'recruitment'
+        - The manager can see all the answers
+        - The recruitment officer can see answers from survey type 'recruitment' unrestricted or in restricted users
+        - The recruitment interviewers can send surveys to applicants and read their answers when they are set as
+        interviewer for these applicants or the job they apply to.
+    -->
+        <!--special rights for recruitment manager on recruitment surveys-->
+        <record id="survey_user_input_rule_recruitment_manager" model="ir.rule">
+            <field name="name">Survey user input: recruitment manager: all</field>
```

Comment:
Would be more precise (this is not 'all', just all related to recruitment surveys)
```suggestion
            <field name="name">Survey user input: recruitment manager: all recruitment</field>
```
If we update naming, to propagate to other rules :)